tests/testsuite/rustc.rs

```diff
@@ -736,21 +708,36 @@ fn rustc_with_print_cfg_multiple_targets() {
 
     p.cargo("rustc -Z unstable-options --target x86_64-pc-windows-msvc --target i686-unknown-linux-gnu --print cfg")
         .masquerade_as_nightly_cargo(&["print"])
-        .with_stdout_contains("debug_assertions")
-        .with_stdout_contains("target_arch=\"x86_64\"")
-        .with_stdout_contains("target_endian=\"little\"")
-        .with_stdout_contains("target_env=\"msvc\"")
-        .with_stdout_contains("target_family=\"windows\"")
-        .with_stdout_contains("target_os=\"windows\"")
-        .with_stdout_contains("target_pointer_width=\"64\"")
-        .with_stdout_contains("target_vendor=\"pc\"")
-        .with_stdout_contains("windows")
-        .with_stdout_contains("target_env=\"gnu\"")
-        .with_stdout_contains("target_family=\"unix\"")
-        .with_stdout_contains("target_pointer_width=\"32\"")
-        .with_stdout_contains("target_vendor=\"unknown\"")
-        .with_stdout_contains("target_os=\"linux\"")
-        .with_stdout_contains("unix")
+        .with_stdout_data(str![[r#"
```

Comment:
The order is not so importnat, should we use `.unordered()` and just one `...` instead?

Reply:
Ok, this also works and make sense to me!